test/types/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Sorry i thought i commented on this earlier...why can't you just use the same tsconfig as the repo?

Reply:
It was necessary when I had a manual index.ts (instead of generated one), but might not be now! Let's see what happens :-)